### PR TITLE
Braun-Blanquet similarity

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#349 
#### What does this implement/fix? Explain your changes.
- Braun-Blanquet similarity added

### CODE REVIEW

File: Document/Document.ipynb [hunk 1]
```diff
@@ -14029,7 +14069,11 @@
     "\n",
     "<blockquote id=\"ref80\">80- R. W. Hamming, \"Error detecting and error correcting codes,\" The Bell system technical journal, vol. 29, no. 2, pp. 147-160, 1950.</blockquote>\n",
     "\n",
-    "<blockquote id=\"ref81\">81- S. S. Choi, S. H. Cha, and C. C. Tappert, \"A survey of binary similarity and distance measures,\" Journal of systemics, cybernetics and informatics, vol. 8, no. 1, pp. 43-48, 2010.</blockquote>\n"
+    "<blockquote id=\"ref81\">81- S. S. Choi, S. H. Cha, and C. C. Tappert, \"A survey of binary similarity and distance measures,\" Journal of systemics, cybernetics and informatics, vol. 8, no. 1, pp. 43-48, 2010.</blockquote>\n",
+    "\n",
+    "<blockquote id=\"ref82\">82- Josias Braun-Blanquet. Plant Sociology: The Study of Plant Communities. McGraw-Hill Book Company, New York, 1932.</blockquote>\n",
```
Comment (sepandhaghighi) [inline Document/Document.ipynb:14074]: @alirezazolanvari Would you please check the format of the references? (82, 83)
Then I will update `README.md`

Comment (alirezazolanvari) [reply Document/Document.ipynb:14074]: @sepandhaghighi the IEEE style bibliography of them are as follows. I couldn't find any other version of 83 for referencing.

"<blockquote id=\"ref82\">82- J. Braun-Blanquet, \"Plant sociology. The study of plant communities,\" <i>Plant sociology. The study of plant communities. First ed.</i>, 1932.</blockquote>\n",
"\n",
"<blockquote id=\"ref83\">83- C. C. Little, \"Abydos Documentation,\" 2018.</blockquote>\n"

Comment (sepandhaghighi) [reply Document/Document.ipynb:14074]: @alirezazolanvari 
Take a look at this [link](https://zenodo.org/record/3603514#.YvOR03ZByUk)
Zenodo gives you the option of changing the citation format.

Comment (alirezazolanvari) [reply Document/Document.ipynb:14074]: I took a look but the provided citation in the IEEE style does not look correct. I suggest to use the google scholar format. I found the citation for its 2020 version.

"<blockquote id=\"ref83\">83- C. C. Little, \"Abydos Documentation,\" 2020.</blockquote>\n"

**Comment by sepandhaghighi** [comment]
@alirezazolanvari @sadrasabouri Please review this PR 🚀

**Review by alirezazolanvari — APPROVED** [review]
🤜🤛

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**